### PR TITLE
[docs][ybm] Add GCP CMK

### PR DESCRIPTION
**Pull request by ddhodge**
Add GCP CMK option

CLOUDGA-13925

@netlify /preview/yugabyte-cloud/cloud-basics/create-clusters/create-single-region/#security

### CODE REVIEW

File: docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-alerts.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ YugabyteDB monitors the health of your clusters based on [cluster alert](#cluste
 | :----- | :---- | :---- |
 | Healthy | No alerts<br/>[Fewer than 34% of nodes down](#fix-nodes-reporting-as-down-alerts) | <br/>Info |
 | Needs Attention | [Node free storage](#fix-storage-alerts)<br/>[More than 34% of nodes down](#fix-nodes-reporting-as-down-alerts)<br/>[Memory Utilization](#fix-memory-alerts)<br/>[YSQL Connections](#fix-ysql-connection-alerts)<br/>[CPU Utilization](#fix-cpu-alerts) | Warning or Severe<br/>Warning<br/>Warning or Severe<br/>Warning or Severe<br/>Warning or Severe
-| Unhealthy | [More than 66% of nodes down](#fix-nodes-reporting-as-down-alerts) | Severe |
+| Unhealthy | [More than 66% of nodes down](#fix-nodes-reporting-as-down-alerts)<br/>[CMK unavailable](#fix-cmk-unavailable-alerts) | Severe |
```
Comment (gurushant-yb) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-alerts.md:59]: It's severity level is `Warning` not `Severe`

**Review by gurushant-yb — APPROVED** [review]
LGTM with nit

File: docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters/include-security-settings.md [hunk 2]
```diff
@@ -6,15 +6,22 @@ private = true
 
 ### Security
 
-In addition to the volume encryption that YugabyteDB Managed uses to encrypt your data, you can enable YugabyteDB [encryption at rest](../../../cloud-secure-clusters/managed-ear/) (EAR) for clusters. When enabled, your YugabyteDB cluster (including backups) is encrypted using a customer managed key (CMK) residing in a cloud provider Key Management Service (KMS). (Currently, only [AWS KMS](https://docs.aws.amazon.com/kms/) is supported.)
+In addition to the volume encryption that YugabyteDB Managed uses to encrypt your data, you can enable YugabyteDB [encryption at rest](../../../cloud-secure-clusters/managed-ear/) (EAR) for clusters. When enabled, your YugabyteDB cluster (including backups) is encrypted using a customer managed key (CMK) residing in a cloud provider Key Management Service (KMS).
 
 You must be signed in as an Admin user to enable EAR. <!--You can also enable EAR for a cluster after the cluster is created.-->
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters/include-security-settings.md:11]: Maybe include a link to admin user .

**Review by aishwarya24 — APPROVED** [review]
LGTM

**Review by aishwarya24 — APPROVED** [review]
LGTM

**Pull request merged**